config.yml

```diff
@@ -83,5 +83,5 @@ params:
     js_bundle_hash:     "sha384-3pZoGeuGuppkOcTDkPFSq3eCeByOZjj3CQ2PRI7hTL8gCUSPno1ZKZo2yeta2U1D"
     popper:             "https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.1/dist/umd/popper.min.js"
     popper_hash:        "sha384-SR1sx49pcuLnqZUnnPwx6FCym0wLsk5JZuNx2bPPENzswTNFaQU1RDvt3wT4gWFG"
-    focus_visible:      "https://cdn.jsdelivr.net/npm/focus-visible@5.1.0/dist/focus-visible.min.js"
+    focus_visible:      "https://cdn.jsdelivr.net/npm/focus-visible@5.2.0/dist/focus-visible.min.js"
```

Comment:
If docs are using bundled JS, why do we use this? In "Getting Started" docs, maybe?

Reply:
Yes you're the author of the callout in introduction. In case project don't use boosted JS

```
{{< callout warning >}}
#### Required script

Boosted includes [WICG's `:focus-visible` polyfill](https://github.com/WICG/focus-visible) to ensure an enhanced focus visibility for keyboard users while shutting down focus styles on active state.
However, if you don't need or want to use Boosted's JavaScript files, you'll still need to use the polyfill.
```html
<script src="{{< param "cdn.focus_visible" >}}" integrity="{{< param "cdn.focus_visible_hash" >}}" crossorigin="anonymous"></script>
```
{{< /callout >}}
```

Reply:
It is used in Getting Started docs indeed:

```js
<script src="{{< param "cdn.focus_visible" >}}" integrity="{{< param "cdn.focus_visible_hash" >}}" crossorigin="anonymous"></script>```